### ISSUE TITLE
AAC afterburner causes output to stop

### ISSUE THREAD

**Issue by autonarcosis** [savonet/liquidsoap#747]
**Describe the bug**
Setting the AAC afterburner to true will cause LS to drop the Icecast connection after about 24-30 hours, it will stop logging any log file output.  The telnet harbor still functions and LS appears to respond to all the commands as if it was working properly.

**To Reproduce**
`output.icecast(%fdkaac(afterburner=true, samplerate=48000, bitrate=64, aot="mpeg2_he_aac", transmux="adts")`

**Expected behavior**
Continue to function as normal.

**Version details**
 - OS: Fedora 29
 - Version 1.3.4 and 1.3.6

**Install method**
opam

**Comment by toots** [comment]
Thanks for this report. To debug further, it would be interesting to know the source of your `fdk-aac` library -- there are a couple of forks around I believe.

Also of use could be a `gdb` trace of the process when the bug is occurring. Are you familiar with the process?

**Comment by autonarcosis** [comment]
Fedora's rpm of fdk-aac is v0.1.6 in the rpm-fusion-nonfree. 
I just did an ldd on the liquidsoap binary.  It appears to be using an fdk-acc.so in /usr/local/lib/ with a 2016 date.  I also found remnants of v1.2.0 and v1.3 when I compiled it outside opam. I must have downloaded and compiled the fdk-aac years ago and forgot about it. I thought opam may have been downloading it and compiling it itself.
I'll download this https://github.com/mstorsjo/fdk-aac and update it and test it.  Pretty sure this is what I used in 2016.
